src/main/proto/command.proto

```diff
@@ -115,4 +115,9 @@ message CommandSubscription {
 
     /* The unique identifier of the component instance subscribing */
     string client_id = 4;
+
+    /* A number that represents the client's relative load capacity compared to other clients.
+       This information is interpreted by Axon Server in relation to the other connected nodes' values.
+       Used to balance the dispatching of commands. If set to 0, Axon Server consider 100 as default value.*/
```

Comment:
Nit: "If set to 0, Axon Server consider 100 as default" Parse error near line 1.
Does this mean that, if set to a non-zero value, 100 is _not_ the default? Or do you mean "0 is interpreted as 100"?

Reply:
I mean that AxonServer interprets 0 as 100. Do you want to suggest to me how to rephrase it in order to be clearer?